### DISCUSSION TITLE
How should I represent a space with an inlet and an outlet in OMEdit?

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10614] [Q&A]
Hello!

How should I represent a space with an inlet and an outlet in OMEdit?

**Comment by casella** [comment]
Do you mean a closed, isolated, well-mixed control volume containing a fluid?

**Reply by ZackeyZAKI** [reply]
Thank you for answering.

I am imagining a typical room with inlet and outlet for air conditioning.

**Comment by casella** [comment]
You can look up examples in the Buildings library that deal with that case. You can install it easily with the [Package Manager](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omedit.html#omedit-install-library-label).

**Reply by ZackeyZAKI** [reply]
Thank you for answering. I'll take a look at the example.
